OpenRA.Game/Selection.cs

```diff
@@ -57,6 +57,15 @@ public void Combine(World world, IEnumerable<Actor> newSelection, bool isCombine
 				}
 			}
 
+			foreach (var a in newSelection)
+				foreach (var sel in a.TraitsImplementing<INotifySelected>())
+					sel.Selected(a);
+			foreach (var ns in world.WorldActor.TraitsImplementing<INotifySelection>())
+				ns.SelectionChanged();
+
+			if (world.IsGameOver)
```

Comment:
I don't know. I only wanted to prevent unit voices from being played. I didn't want to change anything else about selections.